v1/lib/rename.go

```diff
@@ -0,0 +1,116 @@
+package lib
```

Comment:
why'd we drop `internal`? doesn't that mean other libraries can vendor `v1/lib` which could mess with our versioning?

Reply:
Its impossible to import an `internal` package from a sub-package of the project from another part of the project.